packages/varlet-ui/src/picker/Picker.vue

```diff
@@ -149,6 +150,14 @@ export default defineComponent({
 
     let prevIndexes: number[] = []
 
+    watch(
```

Comment:
maybe `computed` is a better choice

Reply:
okay~I agree.
I'm will make changes to this section of the code tonight, based on your advice.

Reply:
done-.-